flutter/lib/src/widgets_binding_observer.dart

```diff
@@ -22,11 +25,14 @@ import '../sentry_flutter.dart';
 class SentryWidgetsBindingObserver with WidgetsBindingObserver {
   SentryWidgetsBindingObserver({
     Hub? hub,
+    bool Function()? isNavigatorObserverCreated,
```

Comment:
Could we move this in third place? Was confused at first why it is never provided, only to see that it is optional while the other named parameter is not. I'm assuming we only have this for testing purposes anyway?